src/utils/application.ts

```diff
@@ -0,0 +1,75 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Form, RetrievedApplication, State } from '@/types/application.types';
+
+function formatApplicationState(state: State) {
```

Comment:
can you add a unit test for this method? If you want you can have that in a separate PR after this is merged